__tests__/app.test.js

```diff
@@ -148,109 +148,245 @@ describe("/api/reviews/:review_id/comments", () => {
+      });
+  });
+});
+
+describe("/api/users", () => {
+  test("GET 200 - responds with array of users", () => {
     return request(app)
-    .post("/api/reviews/badString/comments")
-    .send(newComment)
-    .then(res => {
-        expect(res.body.msg).toBe('Invalid data type')
-    })
+      .get("/api/users")
+      .then((res) => {
+        const users = res.body.users;
+        expect(users.length).toBe(4);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
   });
 });
 
-describe('/api/users', () => {
-  test('GET 200 - responds with array of users', () => {
+describe("/api/reviews?sort_by=title", () => {
+  test("GET 200 - responds with array of reviews sorted by title and descending (defaulted)", () => {
     return request(app)
-    .get("/api/users")
-    .then(res => {
-      const users = res.body.users
-      expect(users.length).toBe(4)
-      users.forEach((user) => {
-        expect(user).toMatchObject({
-          username: expect.any(String),
-          name: expect.any(String),
-          avatar_url: expect.any(String)
-        })
-      })
-    })
+      .get("/api/reviews?sort_by=title")
+      .expect(200)
+      .then((res) => {
+        const reviews = res.body.reviews;
+        expect(reviews.length).toBe(13);
+        reviews.forEach((element) => {
+          expect(element).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+        expect(reviews).toBeSortedBy("title", { descending: true });
+      });
+  });
+  test("GET 200 - responds with array of reviews sorted by comment count and descending (defaulted)", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=comment_count")
+      .expect(200)
+      .then((res) => {
+        const reviews = res.body.reviews;
+        expect(reviews.length).toBe(13);
+        reviews.forEach((element) => {
+          expect(element).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+        expect(reviews).toBeSortedBy("comment_count", { descending: true });
+      });
```

Comment:
☝️ This test does the same as the test above as you are just testing a different column. Is the first test passes, the second one will pass as well :)